blocks/library/paragraph/index.js

```diff
@@ -111,7 +111,7 @@ registerBlockType( 'core/paragraph', {
 				</BlockControls>
 			),
 			focus && (
-				<InspectorControls key="inspector">
+				<InspectorControls key="inspector" identifier="inspector-paragraph">
```

Comment:
What is `identifier`?

Reply:
It was part of an experiment to solve the slot-fill keys problem. And I missed its removal it was corrected.